app/models/user.py

```diff
@@ -0,0 +1,214 @@
+import re
+import uuid
+from typing import Optional
+
+from pydantic import BaseModel, validator
+from sqlmodel import Field, Relationship, SQLModel
+
+
+class DiscordModel(SQLModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
```

Comment:
Is it a good idea to make all these fields nullable? Why not just make the reference to this table nullable instead?

Reply:
Probably not